gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -206,54 +224,66 @@ def validate(self, parsed_args, client=None):
         not parsed_args.sharding_config_path.strip()):
       raise ValueError(
           '--sharding_config_path must point to a valid config file.')
-    # Ensuring (not) existence of output tables is aligned with --append value.
-    if parsed_args.output_table:
-      if (parsed_args.output_table !=
-          bigquery_util.get_table_base_name(parsed_args.output_table)):
-        raise ValueError(('Output table cannot contain "{}" we reserve this  '
+
+    if not client:
+      credentials = GoogleCredentials.get_application_default().create_scoped(
+          ['https://www.googleapis.com/auth/bigquery'])
+      client = bigquery.BigqueryV2(credentials=credentials)
+    if not parsed_args.output_table:
+      raise ValueError('--output_table must have a value.')
+    self._validate_output_tables(
+        client, parsed_args.output_table,
+        parsed_args.sharding_config_path, parsed_args.append)
+
+    if parsed_args.sample_lookup_optimized_output_table:
```

Comment:
We should make sure sample_lookup_optimized_output_table does not equal output_table.

Reply:
Thanks for spotting this corner case!